test/plots/scatterPlotTests.ts

```diff
@@ -19,11 +19,17 @@ describe("Plots", () => {
             .y((d: any) => d.y, yScale);
       });
 
+      afterEach(function() {
+        if (this.currentTest.state === "passed") {
```

Comment:
Yeah, if the test fails, you won't be able to inspect the DOM and understand what went wrong.